### ISSUE TITLE
Home is just a white screen after logging in on Linux Mint

### ISSUE THREAD

**Issue by watercolorhearts** [danielchatfield/trello-desktop#48]
Not sure what is causing it, it seems to have logged in successfully but the Home screen is blank.

**Comment by coroboX** [comment]
Yeah, the same trouble. Cursor is changing arrow/pointer when mouseover something, but only white field below
UPD: after second start problem disappeared
